src/app/containers/Brand/index.jsx

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import Brand from '@bbc/psammead-brand';
-import { ServiceContextConsumer } from '../../contexts/ServiceContext';
+import { ServiceContext } from '../../contexts/ServiceContext';
 
-const BrandContainer = () => (
-  <ServiceContextConsumer>
-    {({ brandName }) => <Brand brandName={brandName} />}
-  </ServiceContextConsumer>
-);
+const BrandContainer = () => {
+  const { brandName, brandSVG } = useContext(ServiceContext);
```

Comment:
`containers/Brand/index.test.jsx` needs to have a context provider as the svg isn't showing in the snapshots